### PR TITLE
Pull Request for Issue1958: Main scaler errors when moving between 'Continuous' and 'SingleShot'

### PR DESCRIPTION
**Pull request by helfrij**
- Removed the class variables for the scans per buffer and the total scans from the AcquisitionControl, they are not used and could be the potential source of some problems.
- Reimplemented updateValue() so that the control's value is only updated when a move is not in progress.

### CODE REVIEW

**Comment by helfrij** [comment]
Hey @XuetaoLiu-CLS, would you take a look at this pull request?

File: source/beamline/CLS/CLSSIS3820ScalerAcquisitionMode.cpp [hunk 1]
```diff
@@ -17,9 +17,6 @@ CLSSIS3820ScalerAcquisitionMode::CLSSIS3820ScalerAcquisitionMode(const QString &
 	numberOfScansPerBufferControl_ = 0;
 	startScanControl_ = 0;
 
-	singleShotScanCountValue_ = 1;
-	singleShotNumberOfScansPerBufferValue_ = 1;
```
Comment (XuetaoLiu-CLS) [inline source/beamline/CLS/CLSSIS3820ScalerAcquisitionMode.cpp]: Great catch! This should be removed for sure :100:

**Comment by XuetaoLiu-CLS** [comment]
Looks good. Great code optimization.

Time Estimate: 10 minutes.

**Comment by dretrex** [comment]
Looks good.
Time Estimate: 3 minutes

**Pull request merged**